app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -1,20 +1,21 @@
 package com.example.drawingactivity
 
+import android.graphics.Bitmap
 import android.graphics.Color
+import androidx.lifecycle.ViewModel
 
 /**
  * A viewModel that controls the colors set in multiple views
  * primaryColor sets the primary color for the app
  * ViewStates is used in the lambda onUpdate to update views in whatever activity it is invoked from
  * cyclePrimaryColor is the internal function that sets a new Primary Color by selecting from a list
  */
-class CustomColorViewModel(
-
-) {
+class CustomColorViewModel : ViewModel() {
     var onUpdate: ((ViewState) -> Unit)? = null
 
     data class ViewState(
         var primaryColor: Int,
+        var viewModelDrawnBitmap: Bitmap?,
```

Comment:
Why does this have to be named `viewModelDrawn`? The view model didn't draw it. Why not just `currentBitmap`?

Reply:
Adjusted it to be currentBitmap instead of viewModelDrawnBitmap